imagetest/README.md

```diff
@@ -79,3 +79,13 @@ It is suggested to start by copying an existing test package.
 From the root of this repository:
 
     $ docker build -t cloud-image-tests -f imagetest/Dockerfile .
+
+## What is being tested ##
+
+The tests are a combination of various types - end to end tests on certain
+software components, image validations and feature validations, etc. The
+collective whole represents the quality assurance bar for releasing a [supported
```

Comment:
We need to clarify this. (and link to somewhere)
Obviously, we can only say this against the images that Google produces. We do list those here: https://cloud.google.com/compute/docs/images/os-details under the Built by section

Reply:
sorry, what should we clarify?

Reply:
Oh- the link... [supported GCE Images][gce-images] is not a thing.